.github/workflows/gradle-build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        java: [11, 17]
+        java: [17]
```

Comment:
```suggestion
        java: [17, 21]
```

Reply:
@jo-elimu This needs Android Gradle build tool 8.5 and up.
Let me upgrade that first.